### ISSUE TITLE
Does code achieve benchmarks

### ISSUE THREAD

**Issue by AlexZhou1995** [dnddnjs/feudal-montezuma#2]
Hi，does this code achieve the benchmarks given in the paper？

**Comment by dnddnjs** [comment]
No not yet. It is under development. If this code achieves the benchmarks, I will write in README😀

**Comment by Yanzzzzz** [comment]
The performance in the paper is really hard to achieve, I've tried many time but failed. 

According another [implementation ](http://www.dshersh.com/portfolio/implementing-feudal-networks/), they have the same problem. 

Maybe we can discuss about some technique details ?

Anyway, wish you luck, and looking forward for your implementation.

**Comment by dnddnjs** [comment]
@Yanzzzzz Yeah, I couldn't find implementation which reproduces the performance of the paper.
Definitely I want to discuss the details about implementation!
